src/results/generator.jl

```diff
@@ -35,13 +35,13 @@ Note: the node number is an empty string if evaluating a single `Site`.
 
 Generator results:
 - `size_kw` Optimal generator capacity
-- `lifecycle_fixed_om_cost` Lifecycle fixed operations and maintenance cost in present value, after tax
-- `year_one_fixed_om_cost` fixed operations and maintenance cost over the first year
-- `lifecycle_variable_om_cost` Lifecycle variable operations and maintenance cost in present value, after tax
-- `year_one_variable_om_cost` variable operations and maintenance cost over the first year
-- `lifecycle_fuel_cost` Lifecycle fuel cost in present value, after tax
-- `year_one_fuel_cost` Fuel cost over the first year
-- `fuel_used_gal` Gallons of fuel used in each year
+- `lifecycle_fixed_om_cost_after_tax` Lifecycle fixed operations and maintenance cost in present value, after tax
+- `year_one_fixed_om_cost_after_tax` fixed operations and maintenance cost over the first year, not considering tax benefits
+- `lifecycle_variable_om_cost_after_tax` Lifecycle variable operations and maintenance cost in present value, after tax
+- `year_one_variable_om_cost` variable operations and maintenance cost over the first year, not considering tax benefits
```

Comment:
we have some Financial outputs with `_before_tax` but we don't use `_before_tax` here. We should be consistent with the `_after_tax` and `_before_tax` results. I prefer removing `_before_tax` but either way works.